### PR TITLE
feat(plasma-tokens): Add `sbermarketVinlab` theme

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.459.4574698184.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.459.4574698184.xml)  
[color_sberHealth_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.459.4574698184.swift)  
[color_sberHealth_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.459.4574698184.kt)  
[color_sberHealth_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.459.4574698184.ts)  
[color_sbermarket_business_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.459.4574698184.swift)  
[color_sbermarket_business_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.459.4574698184.kt)  
[color_sbermarket_business_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.459.4574698184.ts)  
[color_sbermarket_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.459.4574698184.swift)  
[color_sbermarket_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.459.4574698184.kt)  
[color_sbermarket_metro_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.459.4574698184.swift)  
[color_sbermarket_metro_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.459.4574698184.kt)  
[color_sbermarket_metro_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.459.4574698184.ts)  
[color_sbermarket_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.459.4574698184.ts)  
[color_sbermarket_selgros_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.459.4574698184.swift)  
[color_sbermarket_selgros_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.459.4574698184.kt)  
[color_sbermarket_selgros_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.459.4574698184.ts)  
[color_sbermarket_vinlab_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_ios-swift--canary.459.4574698184.swift)  
[color_sbermarket_vinlab_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_kotlin--canary.459.4574698184.kt)  
[color_sbermarket_vinlab_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_react-native--canary.459.4574698184.ts)  
[color_sberprime_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.459.4574698184.swift)  
[color_sberprime_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.459.4574698184.kt)  
[color_sberprime_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.459.4574698184.ts)  
[PlasmaTokensColor--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.459.4574698184.swift)  
[shadow_sbermarket_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.459.4574698184.ts)  
[typo_mage_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.459.4574698184.swift)  
[typo_mage_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.459.4574698184.kt)  
[typo_mage_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.459.4574698184.ts)  
[typo_plasma_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.459.4574698184.swift)  
[typo_plasma_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.459.4574698184.kt)  
[typo_plasma_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.459.4574698184.ts)  
[typo_ruler_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.459.4574698184.swift)  
[typo_ruler_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.459.4574698184.kt)  
[typo_ruler_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.459.4574698184.ts)  
[typo_sage_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.459.4574698184.swift)  
[typo_sage_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.459.4574698184.kt)  
[typo_sage_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.459.4574698184.ts)  
[typo_sbermarket_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.459.4574698184.swift)  
[typo_sbermarket_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.459.4574698184.kt)  
[typo_sbermarket_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.459.4574698184.ts)  
[typo_soulmate_ios-swift--canary.459.4574698184.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.459.4574698184.swift)  
[typo_soulmate_kotlin--canary.459.4574698184.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.459.4574698184.kt)  
[typo_soulmate_react-native--canary.459.4574698184.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.459.4574698184.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.173.0-canary.459.4574698184.0
  npm install @salutejs/plasma-core@1.107.0-canary.459.4574698184.0
  npm install @salutejs/plasma-hope@0.31.0-canary.459.4574698184.0
  npm install @salutejs/plasma-icons@1.136.0-canary.459.4574698184.0
  npm install @salutejs/plasma-temple@1.148.0-canary.459.4574698184.0
  npm install @salutejs/plasma-tokens@1.44.0-canary.459.4574698184.0
  npm install @salutejs/plasma-ui@1.180.0-canary.459.4574698184.0
  npm install @salutejs/plasma-web@1.198.0-canary.459.4574698184.0
  npm install @salutejs/plasma-cy-utils@0.51.0-canary.459.4574698184.0
  npm install @salutejs/plasma-sb-utils@0.105.0-canary.459.4574698184.0
  npm install @salutejs/plasma-tokens-android@2.51.0-canary.459.4574698184.0
  npm install @salutejs/plasma-tokens-ios-swift@2.51.0-canary.459.4574698184.0
  # or 
  yarn add @salutejs/plasma-b2c@1.173.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-core@1.107.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-hope@0.31.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-icons@1.136.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-temple@1.148.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-tokens@1.44.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-ui@1.180.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-web@1.198.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-cy-utils@0.51.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-sb-utils@0.105.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-tokens-android@2.51.0-canary.459.4574698184.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.51.0-canary.459.4574698184.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-459/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-459/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-459/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-459/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-459/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-459/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-459/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by gibushnev — APPROVED** [review]

File: packages/plasma-tokens/data/themes/sbermarketVinlab.json [hunk 1]
```diff
@@ -0,0 +1,653 @@
+{
+    "config": {
+        "name": "sbermarketVinlab",
```
Comment (Yeti-or) [inline packages/plasma-tokens/data/themes/sbermarketVinlab.json:3]: Решили не добавлять - или __ ?

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-tokens/data/themes/sbermarketVinlab.json (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-tokens/data/themes/sbermarketVinlab.json:3]: да, сейчас сборка из-за этого ломается (не может раздулпить в некотором месте, но это легко правится), и решил, что это знак :D Я тебе в телегу писал с этим вопросом

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-459/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-459/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-459/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-459/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-459/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-459/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-459/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-459/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-459/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-459/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-459/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-459/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-459/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-459/temple-storybook/

**Pull request merged**